docs/config.md

```diff
@@ -0,0 +1,200 @@
+# Configuring Password Policy
+
+* [Setting up Password Policy](#setting-up-password-policy)
```

Comment:
This link is not working. Can you please fix this as well

Reply:
Fixed.